app/views/support/home/feature_flags.html.erb

```diff
@@ -0,0 +1,37 @@
+<%- title = t('page_titles.support.feature_flags') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <% breadcrumbs([{ 'Support home' => support_home_path },
+                   title,
+                 ]) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Feature</th>
+          <th scope="col" class="govuk-table__header">Status</th>
+        </tr>
+      </thead>
+
+      <tbody class="govuk-table__body">
+        <% FeatureFlag::FEATURES.sort.each do |feature| %>
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header"><%= feature %></th>
```

Comment:
```suggestion
            <th scope="row" class="govuk-table__header"><%= feature.humanize %></th>
```

Reply:
So that it's a bit easier for non-devs to read ;)

Reply:
I'd think i'd rather leave this as is for now. As I don't want it get lost in translation as it's the actual value and we don't have an associated description or look up value. If it gets confusing or misleading I think we can add something then